arm-redis/2016-04-01/examples/RedisCacheForceReboot.json

```diff
@@ -12,6 +12,7 @@
   "responses": {
     "200": {
       "body": {
+        "Message": "reboot operation enqueued"
```

Comment:
I added this in the example due to an unfortunate fact that it *is* returning pascal case on the wire today.
We did not formally document this member before the exercise with examples. In fact when we reviewed this REST API we did not document any response body. Perhaps the best thing to do here is document the API as returning an empty response, and treat the fact that RP returns a Message property as a bug?